uber/configspec.ini

```diff
@@ -63,6 +63,12 @@ prereg_request_hotel_info_duration = integer(default=0)
 # during preregistration, False to hide the "Confirm Email" field altogether.
 prereg_confirm_email_enabled = boolean(default=False)
 
+# This is the secret link that will be emailed to any attendee that requested
+# hotel booking info during preregistration.
+# NOTE: because this link should be kept secret, it should NEVER be checked
+# into github. The default listed here is merely a placeholder.
+prereg_hotel_info_link = string(default='http://magfest.org/bookhotel/')
```

Comment:
Need to make sure the secrets file on `mcp` is updated, see https://github.com/magfest/production-config/issues/234.